### PR TITLE
[stable/jenkins] DNS Zone customization

### PR DESCRIPTION
**Pull request by tomerleib**
Signed-off-by: Tomer Leibovich <tomer.leibovich@gmail.com>

#### What this PR does / why we need it:
Allows better FQDN resolving in case the K8s cluster was created with different DNS zone, other than `cluster.local`
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x ] Chart Version bumped
- [x] Variables are documented in the README.md
- [x ] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/jenkins/values.yaml [hunk 1]
```diff
@@ -8,6 +8,10 @@
 # nameOverride:
 # fullnameOverride:
 
+# For FQDN resolving of the master service. Change this value to match your existing configuration.
+# ref: https://github.com/kubernetes/dns/blob/master/docs/specification.md
+# clusterZone: "cluster.local"
```
Comment (torstenwalter) [inline stable/jenkins/values.yaml]: you need to remove the comment otherwise we don't have a default value

**Review by torstenwalter — CHANGES_REQUESTED** [review]

File: stable/jenkins/values.yaml (hunk 1 above)
Comment (tomerleib) [reply stable/jenkins/values.yaml]: fixed.

**Comment by torstenwalter** [comment]
You need to sign the last commit:
```
git commit --amend -s
git push --force
```

**Comment by torstenwalter** [comment]
@tomerleib Thank you for the contribution!

**Comment by tomerleib** [comment]
@torstenwalter my pleasure :)

**Comment by maorfr** [comment]
well done @tomerleib

**Pull request merged**